TestExperimentDataIntegration.test_block_for_results fails with AssertionError: 'DONE' != <ExperimentStatus.DONE: 'experiment jobs and analysis have successfully run'>

### Informations

- **Qiskit Experiments version**: main branch:

```
$ git log -1 --oneline
ff272ff (HEAD -> main, origin/main, origin/HEAD) Use fixed simulator seeds in the tutorials
```

- **Python version**: 3.8
- **Operating system**: Ubuntu 20.04

### What is the current behavior?

The test fails, at least locally for me. The integration tests aren't yet running in the cron-staging workflow (see #704).

```
$ python -m unittest -v test.database_service.test_experiment_data_integration.TestExperimentDataIntegration.test_block_for_results
test_block_for_results (test.database_service.test_experiment_data_integration.TestExperimentDataIntegration)
test.database_service.test_experiment_data_integration.TestExperimentDataIntegration.test_block_for_results ... FAIL

======================================================================
FAIL: test_block_for_results (test.database_service.test_experiment_data_integration.TestExperimentDataIntegration)
test.database_service.test_experiment_data_integration.TestExperimentDataIntegration.test_block_for_results
----------------------------------------------------------------------
testtools.testresult.real._StringException: Traceback (most recent call last):
  File "/home/osboxes/ibmq/qiskit-experiments/test/database_service/test_experiment_data_integration.py", line 473, in test_block_for_results
    self.assertEqual("DONE", exp_data.status())
  File "/usr/lib/python3.8/unittest/case.py", line 912, in assertEqual
    assertion_func(first, second, msg=msg)
  File "/usr/lib/python3.8/unittest/case.py", line 905, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: 'DONE' != <ExperimentStatus.DONE: 'experiment jobs and analysis have successfully run'>


----------------------------------------------------------------------
Ran 1 test in 91.691s

FAILED (failures=1)
```

### Steps to reproduce the problem

Run the tests. 😄 

### What is the expected behavior?

Tests should pass.

### Suggested solutions

It seems that `assertEqual` should just be changed to `assertIn`.

--- a/test/database_service/test_experiment_data_integration.py
+++ b/test/database_service/test_experiment_data_integration.py
@@ -30,6 +30,7 @@ from qiskit.providers.ibmq.experiment import (
 from qiskit.test import QiskitTestCase
 from qiskit_experiments.database_service import DbExperimentDataV1 as DbExperimentData
 from qiskit_experiments.database_service import DbAnalysisResultV1 as AnalysisResult
+from qiskit_experiments.database_service.db_experiment_data import ExperimentStatus
 from qiskit_experiments.database_service.exceptions import DbExperimentEntryNotFound
 
 from ..decorators import requires_provider, requires_device
@@ -470,7 +471,7 @@ class TestExperimentDataIntegration(QiskitTestCase):
             jobs.append(job)
         exp_data.block_for_results()
         self.assertTrue(all(job.status() == JobStatus.DONE for job in jobs))
-        self.assertEqual("DONE", exp_data.status())
+        self.assertEqual(ExperimentStatus.DONE, exp_data.status())
 
     def _create_experiment_data(self):
         """Create an experiment data."""

